devsetup/Makefile

```diff
@@ -493,6 +494,7 @@ tripleo_deploy: export COMPUTE_CELLS=${EDPM_COMPUTE_CELLS}
 tripleo_deploy: export REGISTRY_USER ?= ${RH_REGISTRY_USER}
 tripleo_deploy: export REGISTRY_PWD ?= ${RH_REGISTRY_PWD}
 tripleo_deploy: export COMPUTE_CEPH_ENABLED=${EDPM_COMPUTE_CEPH_ENABLED}
+tripleo_deploy: export CONFIGURE_HUGEPAGES=${EDPM_CONFIGURE_HUGEPAGES}
```

Comment:
Wondering why we use two different names (with EDPM_ prefix and without) but it seems the precedent is already there :).